### DISCUSSION TITLE
Logic glow

### DISCUSSION THREAD

**Discussion by ZachZent** [tsunamayo/Starship-EVO#4154] [Suggestions]
Just a small idea I thought in the Discord, would it be possible for logic bricks to slightly glow (no emitted light) when "activated/pulsed"? It would be a small visual effect, but it would make things like server rooms look super cool. Assuming of course that it wouldn't be too resource intensive for such a small effect, but who doesn't love blinking server rooms before you fire a rocket launcher at it?

**Comment by Asnro** [comment]
This would also be super useful for debugging logic systems, as currently you'd have to manually link status lights or other mechanisms to each gate to figure out if they activate.

**Comment by pinesh** [comment]
Upvoting! Nice quality of life idea.
